### PR TITLE
Add Readme.md file to To - Do list

### PR DESCRIPTION
**Pull request by Atif0604**
Hey @Ayushparikh-code @chicken-biryani 
This PR will Resolve => Add Readme.md file to To - Do list webapp. #53
Please Review my PR

### CODE REVIEW

File: To-Do List/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ <img src="todo.png" alt="To-Do  Image"/>
+ <br>
+ <ul>
+  <li>To add the Task,Write the Task in the box and click on Add Button<br><br>
+    <img src="todo-add.png" alt="To-Do Add Image"/><br></li><br>
+   <li>If you have completed the task,click on the task to strike-out<br><br>
+    <img src="todo-delete.png" alt="To-Do Strikeout Image"/><br></li><br>
+     <li>To Remove the completed Task,First click on the task to strike-out and then Click on Cleanup<br><br>
+    <img src="todo remove.png" alt="To-Do Remove Image"/><br></li><br>
+</ul>
```
Comment (chicken-biryani) [inline To-Do List/README.md:40]: Hey Atif! It is the Images.
Change the path of the src. 
The link below will help
https://stackoverflow.com/questions/14494747/how-to-add-images-to-readme-md-on-github

File: To-Do List/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+git clone https://github.com/Ayushparikh-code/Web-dev-mini-projects.git
+```
+
+- Go to the directory
+- Run the index.html file
+- Add your tasks According to your wish
+
+<h3> ScreenShots </h3> 
+ <img src="todo.png" alt="To-Do  Image"/>
```
Comment (chicken-biryani) [inline To-Do List/README.md:31]: Change src path here.

File: To-Do List/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ <br>
+ <ul>
+  <li>To add the Task,Write the Task in the box and click on Add Button<br><br>
+    <img src="todo-add.png" alt="To-Do Add Image"/><br></li><br>
```
Comment (chicken-biryani) [inline To-Do List/README.md:35]: Change src path here

File: To-Do List/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+   <li>If you have completed the task,click on the task to strike-out<br><br>
+    <img src="todo-delete.png" alt="To-Do Strikeout Image"/><br></li><br>
```
Comment (chicken-biryani) [inline To-Do List/README.md:37]: Change src path here

File: To-Do List/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+     <li>To Remove the completed Task,First click on the task to strike-out and then Click on Cleanup<br><br>
+    <img src="todo remove.png" alt="To-Do Remove Image"/><br></li><br>
```
Comment (chicken-biryani) [inline To-Do List/README.md:39]: Change src path here.

**Comment by chicken-biryani** [comment]
You can take @yellowberard 's help if you want. He created a similar PR https://github.com/Ayushparikh-code/Web-dev-mini-projects/pull/57

**Comment by Atif0604** [comment]
> You can take @yellowberard 's help if you want. He created a similar PR #57

i saw his code ,he directly uploaded the file in the Readme itself and i was making the Readme in my Vs Code
That's why the url was diff
Hope It's Fine now @chicken-biryani

**Pull request merged**